[2024-12-05 15:34:22] INFO - train: Starting training process...
[2024-12-05 15:34:22] INFO - train: Loading configurations...
[2024-12-05 15:34:22] INFO - train: Using device: cuda
[2024-12-05 15:34:22] INFO - train: Creating data loaders...
[2024-12-05 15:35:05] INFO - train: Setting up models and trainer...
[2024-12-05 15:35:05] INFO - train: Initializing context_aware distillation training...
[2024-12-05 15:35:05] ERROR - train: An error occurred: 'name'
Traceback (most recent call last):
  File "/nas2/jieui/class/deepnlp/project/scripts/train.py", line 64, in main
    teacher_model = model_factory.create_model('bert', model_configs[0])
  File "/nas2/jieui/class/deepnlp/project/src/models/model_factory.py", line 26, in create_model
    base_model = AutoModel.from_pretrained(config['name'])  # 'name' 사용
KeyError: 'name'
